### PR TITLE
#164369336 Create a static footer component

### PR DESCRIPTION
**Pull request by primuse**
#### What does this PR do?
Create a static footer component.
#### How should this be manually tested?
Clone the Repo, run `npm install`, then run `npm test`.
#### What are the relevant pivotal tracker stories?
[#164369336](https://www.pivotaltracker.com/story/show/164369336)
#### Screenshots (if appropriate)
- Laptop/desktop view
laptop_view

- Tablet view
tablet_view

- Mobile view
mobile_view

### CODE REVIEW

File: src/enzyme.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
```
Comment (ozone4real) [inline src/enzyme.js]: You didn't delete this. Please do.

**Review by ozone4real — DISMISSED** [review]

File: src/enzyme.js (hunk 1 above)
Comment (primuse) [reply src/enzyme.js]: Noted. Would do that ASAP

**Review by ozone4real — DISMISSED** [review]
LGTM

**Review by EfeAgare — DISMISSED** [review]

**Review by EfeAgare — APPROVED** [review]

**Review by ozone4real — APPROVED** [review]
LGTM

**Pull request merged**